### ISSUE TITLE
Usually modifications don't get saved.

### ISSUE THREAD

**Issue by Axure** [kenwei/pdf_reader_x#235]
When you save and reopen the document, usually it is just unchanged.

**Comment by alexandrejm** [comment]
I am using Yosemite yet and it seems there is no problem (almost of them,
at least).

I have no crash, no saving problems. But I have disabled the Auto Saving. I
save by myself or when I quit.

By the way, I am studing to move to other app as I intend to update to El
Capitan soon. I didn´t yet because of Outline feature that I use a lot.

This app had everything to be one of the best PDF app in the market, but
unfortunately the updates are not fast.

2015-11-19 5:07 GMT-02:00 Hu Zheng <notifications@github.com>:

**Comment by themissessie** [comment]
yes, I do agree that the file could not be saved, it was so frustrating when all the notes after one lecture could not be found despite me trying to save it like every 10 min manually.  Grateful if the developer can dig into this issue and get it fixed!

**Comment by Axure** [comment]
@alexandrejm, when I disable auto save and save manually, every time it gives a popup that says the document cannot be saved. I suspect that there are some kind of systematic problems (permissions or something?).

El Capitan 10.11.1

**Comment by alexandrejm** [comment]
I guess it is an incompatibility to El Capitan.

I am using Yosemit and have no issues after the latest update. Probably the
developer didn't update to El Capitan yet.

2015-11-20 19:47 GMT-02:00 Hu Zheng <notifications@github.com>:
